### PR TITLE
Enhancements accross application

### PR DESCRIPTION
**Pull request by MarcelGeo**
:hammer:  Fixes:

- ui: :iphone:  action buttons for project

image (10)

- Infinite in full storage banner in case of diskusage > storage

- Overflow wrap for email in ProfileView
image

Feature:
- New Avatars handling:
_without Full Name_
image

_with only one name_
image

_with full name_
image

_example for access_
image

### CODE REVIEW

File: web-app/packages/lib/src/common/components/FullStorageWarningTemplate.vue [hunk 1]
```diff
@@ -31,11 +31,13 @@ import { useUserStore } from '@/main'
 const userStore = useUserStore()
 
 const usage = computed(() =>
-  Math.floor(
-    (userStore.currentWorkspace?.disk_usage /
-      userStore.currentWorkspace?.storage) *
-      100
-  )
+  userStore.currentWorkspace?.disk_usage > userStore.currentWorkspace?.storage
+    ? 0
```
Comment (varmar05) [inline web-app/packages/lib/src/common/components/FullStorageWarningTemplate.vue:35]: should not be 100%?

Comment (MarcelGeo) [reply web-app/packages/lib/src/common/components/FullStorageWarningTemplate.vue:35]: if storage === 0 , this is not possible to make disk_usage / storage. We do not want to show this banner in such edge case.

Comment (varmar05) [reply web-app/packages/lib/src/common/components/FullStorageWarningTemplate.vue:35]: but disk usage can be > storage in other cases, like plan downgrade?

Comment (MarcelGeo) [reply web-app/packages/lib/src/common/components/FullStorageWarningTemplate.vue:35]: yeah, but in these cases -> it is not reasonable to warn user about full storage :)

Comment (varmar05) [reply web-app/packages/lib/src/common/components/FullStorageWarningTemplate.vue:35]: ah, that is only warning, sorry, I missed it completely

**Review by varmar05 — APPROVED** [review]

**Comment by jozef-budac** [comment]
Just check it on lasters RC and it looks good to me :)

**Pull request merged**